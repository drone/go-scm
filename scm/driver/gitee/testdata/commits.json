[
  {
    "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/e3c0ff4d5cef439ea11b30866fb1ed79b420801d",
    "sha": "e3c0ff4d5cef439ea11b30866fb1ed79b420801d",
    "html_url": "https://gitee.com/kit101/drone-yml-test/commit/e3c0ff4d5cef439ea11b30866fb1ed79b420801d",
    "comments_url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/e3c0ff4d5cef439ea11b30866fb1ed79b420801d/comments",
    "commit": {
      "author": {
        "name": "kit101",
        "date": "2021-08-23T23:45:53+08:00",
        "email": "qkssk1711@163.com"
      },
      "committer": {
        "name": "kit101",
        "date": "2021-08-23T23:45:53+08:00",
        "email": "qkssk1711@163.com"
      },
      "message": "add/update/delete/rename\n",
      "tree": {
        "sha": "55d4c682f3e2b276cc40c70e59cebb17dcb1273a",
        "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/git/trees/55d4c682f3e2b276cc40c70e59cebb17dcb1273a"
      }
    },
    "author": {
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "url": "https://gitee.com/api/v5/users/kit101",
      "html_url": "https://gitee.com/kit101",
      "remark": "",
      "followers_url": "https://gitee.com/api/v5/users/kit101/followers",
      "following_url": "https://gitee.com/api/v5/users/kit101/following_url{/other_user}",
      "gists_url": "https://gitee.com/api/v5/users/kit101/gists{/gist_id}",
      "starred_url": "https://gitee.com/api/v5/users/kit101/starred{/owner}{/repo}",
      "subscriptions_url": "https://gitee.com/api/v5/users/kit101/subscriptions",
      "organizations_url": "https://gitee.com/api/v5/users/kit101/orgs",
      "repos_url": "https://gitee.com/api/v5/users/kit101/repos",
      "events_url": "https://gitee.com/api/v5/users/kit101/events{/privacy}",
      "received_events_url": "https://gitee.com/api/v5/users/kit101/received_events",
      "type": "User"
    },
    "committer": {
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "url": "https://gitee.com/api/v5/users/kit101",
      "html_url": "https://gitee.com/kit101",
      "remark": "",
      "followers_url": "https://gitee.com/api/v5/users/kit101/followers",
      "following_url": "https://gitee.com/api/v5/users/kit101/following_url{/other_user}",
      "gists_url": "https://gitee.com/api/v5/users/kit101/gists{/gist_id}",
      "starred_url": "https://gitee.com/api/v5/users/kit101/starred{/owner}{/repo}",
      "subscriptions_url": "https://gitee.com/api/v5/users/kit101/subscriptions",
      "organizations_url": "https://gitee.com/api/v5/users/kit101/orgs",
      "repos_url": "https://gitee.com/api/v5/users/kit101/repos",
      "events_url": "https://gitee.com/api/v5/users/kit101/events{/privacy}",
      "received_events_url": "https://gitee.com/api/v5/users/kit101/received_events",
      "type": "User"
    },
    "parents": [
      {
        "sha": "537575f44a09c57dfc472e26fe067754fd2f9374",
        "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/537575f44a09c57dfc472e26fe067754fd2f9374"
      }
    ]
  },
  {
    "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/537575f44a09c57dfc472e26fe067754fd2f9374",
    "sha": "537575f44a09c57dfc472e26fe067754fd2f9374",
    "html_url": "https://gitee.com/kit101/drone-yml-test/commit/537575f44a09c57dfc472e26fe067754fd2f9374",
    "comments_url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/537575f44a09c57dfc472e26fe067754fd2f9374/comments",
    "commit": {
      "author": {
        "name": "kit101",
        "date": "2021-08-23T23:45:04+08:00",
        "email": "qkssk1711@163.com"
      },
      "committer": {
        "name": "kit101",
        "date": "2021-08-23T23:45:04+08:00",
        "email": "qkssk1711@163.com"
      },
      "message": "change ready\n",
      "tree": {
        "sha": "c5851e358f5c1950ce15772b0cf05941470793bc",
        "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/git/trees/c5851e358f5c1950ce15772b0cf05941470793bc"
      }
    },
    "author": {
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "url": "https://gitee.com/api/v5/users/kit101",
      "html_url": "https://gitee.com/kit101",
      "remark": "",
      "followers_url": "https://gitee.com/api/v5/users/kit101/followers",
      "following_url": "https://gitee.com/api/v5/users/kit101/following_url{/other_user}",
      "gists_url": "https://gitee.com/api/v5/users/kit101/gists{/gist_id}",
      "starred_url": "https://gitee.com/api/v5/users/kit101/starred{/owner}{/repo}",
      "subscriptions_url": "https://gitee.com/api/v5/users/kit101/subscriptions",
      "organizations_url": "https://gitee.com/api/v5/users/kit101/orgs",
      "repos_url": "https://gitee.com/api/v5/users/kit101/repos",
      "events_url": "https://gitee.com/api/v5/users/kit101/events{/privacy}",
      "received_events_url": "https://gitee.com/api/v5/users/kit101/received_events",
      "type": "User"
    },
    "committer": {
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "url": "https://gitee.com/api/v5/users/kit101",
      "html_url": "https://gitee.com/kit101",
      "remark": "",
      "followers_url": "https://gitee.com/api/v5/users/kit101/followers",
      "following_url": "https://gitee.com/api/v5/users/kit101/following_url{/other_user}",
      "gists_url": "https://gitee.com/api/v5/users/kit101/gists{/gist_id}",
      "starred_url": "https://gitee.com/api/v5/users/kit101/starred{/owner}{/repo}",
      "subscriptions_url": "https://gitee.com/api/v5/users/kit101/subscriptions",
      "organizations_url": "https://gitee.com/api/v5/users/kit101/orgs",
      "repos_url": "https://gitee.com/api/v5/users/kit101/repos",
      "events_url": "https://gitee.com/api/v5/users/kit101/events{/privacy}",
      "received_events_url": "https://gitee.com/api/v5/users/kit101/received_events",
      "type": "User"
    },
    "parents": [
      {
        "sha": "7e84b6f94b8d4bfaa051910cc4ce16b73bcffd51",
        "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/7e84b6f94b8d4bfaa051910cc4ce16b73bcffd51"
      }
    ]
  },
  {
    "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/7e84b6f94b8d4bfaa051910cc4ce16b73bcffd51",
    "sha": "7e84b6f94b8d4bfaa051910cc4ce16b73bcffd51",
    "html_url": "https://gitee.com/kit101/drone-yml-test/commit/7e84b6f94b8d4bfaa051910cc4ce16b73bcffd51",
    "comments_url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/7e84b6f94b8d4bfaa051910cc4ce16b73bcffd51/comments",
    "commit": {
      "author": {
        "name": "kit101",
        "date": "2021-08-23T23:41:22+08:00",
        "email": "qkssk1711@163.com"
      },
      "committer": {
        "name": "kit101",
        "date": "2021-08-23T23:41:22+08:00",
        "email": "qkssk1711@163.com"
      },
      "message": "rename\n",
      "tree": {
        "sha": "3dd0fe5f5bcde3966871c8eb2c91d55554dab80f",
        "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/git/trees/3dd0fe5f5bcde3966871c8eb2c91d55554dab80f"
      }
    },
    "author": {
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "url": "https://gitee.com/api/v5/users/kit101",
      "html_url": "https://gitee.com/kit101",
      "remark": "",
      "followers_url": "https://gitee.com/api/v5/users/kit101/followers",
      "following_url": "https://gitee.com/api/v5/users/kit101/following_url{/other_user}",
      "gists_url": "https://gitee.com/api/v5/users/kit101/gists{/gist_id}",
      "starred_url": "https://gitee.com/api/v5/users/kit101/starred{/owner}{/repo}",
      "subscriptions_url": "https://gitee.com/api/v5/users/kit101/subscriptions",
      "organizations_url": "https://gitee.com/api/v5/users/kit101/orgs",
      "repos_url": "https://gitee.com/api/v5/users/kit101/repos",
      "events_url": "https://gitee.com/api/v5/users/kit101/events{/privacy}",
      "received_events_url": "https://gitee.com/api/v5/users/kit101/received_events",
      "type": "User"
    },
    "committer": {
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "url": "https://gitee.com/api/v5/users/kit101",
      "html_url": "https://gitee.com/kit101",
      "remark": "",
      "followers_url": "https://gitee.com/api/v5/users/kit101/followers",
      "following_url": "https://gitee.com/api/v5/users/kit101/following_url{/other_user}",
      "gists_url": "https://gitee.com/api/v5/users/kit101/gists{/gist_id}",
      "starred_url": "https://gitee.com/api/v5/users/kit101/starred{/owner}{/repo}",
      "subscriptions_url": "https://gitee.com/api/v5/users/kit101/subscriptions",
      "organizations_url": "https://gitee.com/api/v5/users/kit101/orgs",
      "repos_url": "https://gitee.com/api/v5/users/kit101/repos",
      "events_url": "https://gitee.com/api/v5/users/kit101/events{/privacy}",
      "received_events_url": "https://gitee.com/api/v5/users/kit101/received_events",
      "type": "User"
    },
    "parents": [
      {
        "sha": "f87d3cca9d85a7e52bb9dadac176817812b76fbd",
        "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/f87d3cca9d85a7e52bb9dadac176817812b76fbd"
      }
    ]
  }
]