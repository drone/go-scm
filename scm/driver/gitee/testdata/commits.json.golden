[
  {
    "Sha": "e3c0ff4d5cef439ea11b30866fb1ed79b420801d",
    "Message": "add/update/delete/rename\n",
    "Author": {
      "Name": "kit101",
      "Email": "qkssk1711@163.com",
      "Date": "2021-08-23T23:45:53+08:00",
      "Login": "kit101",
      "Avatar": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png"
    },
    "Committer": {
      "Name": "kit101",
      "Email": "qkssk1711@163.com",
      "Date": "2021-08-23T23:45:53+08:00",
      "Login": "kit101",
      "Avatar": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png"
    },
    "Link": "https://gitee.com/kit101/drone-yml-test/commit/e3c0ff4d5cef439ea11b30866fb1ed79b420801d"
  },
  {
    "Sha": "537575f44a09c57dfc472e26fe067754fd2f9374",
    "Message": "change ready\n",
    "Author": {
      "Name": "kit101",
      "Email": "qkssk1711@163.com",
      "Date": "2021-08-23T23:45:04+08:00",
      "Login": "kit101",
      "Avatar": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png"
    },
    "Committer": {
      "Name": "kit101",
      "Email": "qkssk1711@163.com",
      "Date": "2021-08-23T23:45:04+08:00",
      "Login": "kit101",
      "Avatar": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png"
    },
    "Link": "https://gitee.com/kit101/drone-yml-test/commit/537575f44a09c57dfc472e26fe067754fd2f9374"
  },
  {
    "Sha": "7e84b6f94b8d4bfaa051910cc4ce16b73bcffd51",
    "Message": "rename\n",
    "Author": {
      "Name": "kit101",
      "Email": "qkssk1711@163.com",
      "Date": "2021-08-23T23:41:22+08:00",
      "Login": "kit101",
      "Avatar": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png"
    },
    "Committer": {
      "Name": "kit101",
      "Email": "qkssk1711@163.com",
      "Date": "2021-08-23T23:41:22+08:00",
      "Login": "kit101",
      "Avatar": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png"
    },
    "Link": "https://gitee.com/kit101/drone-yml-test/commit/7e84b6f94b8d4bfaa051910cc4ce16b73bcffd51"
  }
]